doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -191,17 +197,23 @@ The below set of steps define the compliance, verification and Conformance proce
   * Images uploaded to tenant space
   * External system test data needs identified
   * Owners (NFVI, VNF, PTL, etc) documented
-  * Security Compliance Satisfied (Refer to CNTT Chapter XXXX Security for additional tests, scans, and vulnerabilities validations)
+  * Security Compliance Satisfied (Refer to CNTT Chapter XXXX Security for
```

Comment:
should this state, "(Refer to [RM Chapter 07](../../ref_model/chapters/chapter07.md) for additional security requirements)"  --